### DISCUSSION TITLE
Gant chart or timeline view

### DISCUSSION THREAD

**Discussion by detourne09** [marcusolsson/obsidian-projects#890] [Help]
Hi there

Do you know of any way to add a timeline or gant view to projects?

Thanks

**Comment by Acylation** [comment]
Sorry currently we don't have one but we're on the way of building one. Many improvements focusing on date and datetime are made recently. Please track #6 for any progress.

**Comment by MrEbbinghaus** [comment]
@detourne09 You can make your own using mermaid diagrams:
https://help.obsidian.md/Editing+and+formatting/Advanced+formatting+syntax#Diagram

https://mermaid.js.org/syntax/gantt.html
https://mermaid.js.org/syntax/timeline.html
